components/application-configuration/ui/src/main/resources/PhenoTips/DBConfigurationClass.xml

```diff
@@ -67,7 +67,7 @@ $!value
       <name>dateOfBirthFormat</name>
       <number>5</number>
       <picker>0</picker>
-      <prettyName>Date of birth format</prettyName>
+      <prettyName>$services.localization.render('phenotips.DBConfigurationClass.dateOfBirthFormat')</prettyName>
```

Comment:
You actually did translate that, you just forgot to revert the changes in this file.